packages/grid/_modules_/grid/hooks/root/useEvents.ts

```diff
@@ -199,8 +199,8 @@ export function useEvents(gridRootRef: React.RefObject<HTMLDivElement>, apiRef:
       return () => {
         logger.debug('Clearing all events listeners');
         api.publishEvent(GRID_UNMOUNT);
-        gridRootElem.removeEventListener(GRID_CLICK, onClickHandler, { capture: true });
-        gridRootElem.removeEventListener(GRID_MOUSE_HOVER, onHoverHandler, { capture: true });
+        // gridRootElem.removeEventListener(GRID_CLICK, onClickHandler, { capture: true });
```

Comment:
I guess this is still WIP but if you are removing those can you also remove the constants if they are not needed anymore?